bindings/c/manifoldc.cpp

```diff
@@ -202,6 +202,17 @@ ManifoldManifold *manifold_trim_by_plane(void *mem, ManifoldManifold *m,
   return to_c(new (mem) Manifold(trimmed));
 }
 
+ManifoldCrossSection *manifold_slice(void *mem, ManifoldManifold *m,
+                                     float height) {
+  auto poly = from_c(m)->Slice(height);
+  return to_c(new (mem) CrossSection(poly));
+}
+
+ManifoldCrossSection *manifold_project(void *mem, ManifoldManifold *m) {
+  auto poly = from_c(m)->Project();
+  return to_c(new (mem) CrossSection(poly));
```

Comment:
@geoffder Does this look right?

Reply:
Yes (late I know, already merged). Sorry I've been swamped in trying to wrap up things with my PhD (and will be for a while) so I've been skimming too many of my manifold GitHub email notifications (though I've still been trying to read most of them to stay up to date).

Reply:
No prob! I know how it is with wrapping up a PhD - good luck! It's better on the other side, I promise!